packages/core/tokens/CHANGELOG.md

```
@@ -2,6 +2,12 @@
 
 Our design tokens are the foundations of Nimbus Design System.
 
+## 2025-08-20 `9.3.0`
+
+#### 🎉 New features
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)